docs/tutorial.md

```diff
@@ -9,7 +9,11 @@ This tutorial will walk you through creating and running some simple
 - [Creating a hello world `Task`](#task)
 - [Creating a hello world `Pipeline`](#pipeline)
 
-Before starting this tutorial, please install the [Tekton CLI](https://github.com/tektoncd/cli).
+Before starting this tutorial, please install the [Tekton CLI](https://github.com/tektoncd/cli) and install the latest Tekton release by the following command:
```

Comment:
Instead of hard-coding the kubectl command here, how about linking to the install doc we have for tekton at ./install.md?  The user might need to go through the steps of setting up a cluster as well before starting the tutorial and that would be covered by linking to the existing install doc.